actionview/lib/action_view/railtie.rb

```diff
@@ -37,7 +37,7 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
-    initializer "action_view.collection_caching" do |app|
+    initializer "action_view.collection_caching", after: "action_controller.set_configs" do |app|
```

Comment:
We're in Railties land, the thing that ties all Rails' components together, so I think it's meant to be a little coupling.

But this doesn't force anybody who wants to use Action View to also pull in Action Controller.

And this code should run fine if the specified `:after` initializer isn't there, if I'm reading this code correctly: https://github.com/rails/rails/blob/c1ad19c92fde1c3801a93b59003ba2b266e4128f/railties/lib/rails/initializable.rb#L44.